### PR TITLE
fix [Library], AutoDjAddBottom/Top and add AutoDjAddReplace

### PR DESCRIPTION
**Pull request by Be-ing**
https://bugs.launchpad.net/mixxx/+bug/1887264

### CODE REVIEW

**Comment by uklotzde** [comment]
master or 2.3?

**Comment by Be-ing** [comment]
Whoops, forgot to change the branch target on GitHub.

File: src/library/librarycontrol.h [hunk 1]
```diff
@@ -132,6 +133,7 @@ class LibraryControl : public QObject {
     // Add to Auto-Dj Cueue
```
Comment (uklotzde) [inline src/library/librarycontrol.h:133]: Unrelated typo, let's fix it

Comment (Holzhaus) [reply src/library/librarycontrol.h:133]: For convenience:

```suggestion
    // Add to AutoDJ queue
```

**Comment by uklotzde** [comment]
LGTM. Waiting for CI.

**Pull request merged**